src/main/java/com/github/alexcojocaru/mojo/elasticsearch/RunElasticSearchNodeMojo.java

```diff
@@ -0,0 +1,66 @@
+package com.github.alexcojocaru.mojo.elasticsearch;
+
+import org.apache.maven.plugin.MojoExecutionException;
+
+import java.io.File;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Starts a local instance of ElasticSearch indefinitely.
+ * In order to kill it a CTRL+C has to be made.
+ *
+ * @author gfernandes
+ * @goal run
+ * @execute phase="compile"
+ * @requiresDependencyResolution runtime
+ */
+public class RunElasticSearchNodeMojo extends AbstractStartElasticsearchNodeMojo {
```

Comment:
Please rename to RunElasticsearchNodeMojo, to keep the naming consistent across classes.